.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
+  "singleQuote": true,
+  "printWidth": 120,
+  "semi": false,
+  "arrowParens": "avoid"
+}
```

Comment:
If we go down the route of extracting out an eslint then ideally prettier and it's config would go to...